Source/WTF/wtf/glib/SpanHelpers.h

```diff
@@ -0,0 +1,43 @@
+/*
+ *  Copyright (C) 2024 Igalia S.L.
+ *
+ *  This library is free software; you can redistribute it and/or
+ *  modify it under the terms of the GNU Library General Public
+ *  License as published by the Free Software Foundation; either
+ *  version 2 of the License, or (at your option) any later version.
+ *
+ *  This library is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ *  Library General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Library General Public License
+ *  along with this library; see the file COPYING.LIB.  If not, write to
+ *  the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ *  Boston, MA 02110-1301, USA.
+ */
+
+#pragma once
+
+#include "GRefPtr.h"
+
+namespace WTF {
+
+template <typename T>
+inline std::span<T> asSpan(GBytes* bytes)
```

Comment:
We've been calling these helper functions just "span".

Reply:
Since the type returned by g_bytes_get_data is a uint8, I'd suggest making this a plain (non-template) function that returns uint8, and using span_reinterpret_cast anywhere you need something different. That way, we establish a separation of concerns between documenting bounds and performing type conversion.

Reply:
Thanks, I will update taking both suggestions into account 👍🏼